### PR TITLE
Fabo/1642 proposal header css

### LINKED ISSUE #1642: TableProposal headers misaligned
Description:

screen shot 2018-11-26 at 11 31 56 am

### PR DESCRIPTION
**Pull request by faboweb**
Closes #1642

_Description:_

Headers before:
image
image

Headers now:
image
image

❤️ Thank you!

---

- [ ] Added entries in `CHANGELOG.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer

### CODE REVIEW

**Comment by jbibla** [comment]
title and moniker should be aligned with the left border of the table

File: app/src/renderer/styles/table.css [hunk 1]
```diff
@@ -0,0 +1,104 @@
+.data-table {
```
Comment (sabau) [inline app/src/renderer/styles/table.css]: Nice separtion of concerns

File: app/src/renderer/components/staking/LiValidator.vue [hunk 2]
```diff
@@ -141,50 +141,20 @@ export default {
...
-  overflow: hidden;
-  white-space: nowrap;
-  text-overflow: ellipsis;
-}
-.li-validator__avatar {
-  height: 3rem;
-  width: 3rem;
-  margin: 1rem 0.5rem;
-  border-radius: 50%;
+.li-validator.data-table__row td:nth-child(3):after {
```
Comment (sabau) [inline app/src/renderer/components/staking/LiValidator.vue]: Really minor: I'm thinking if relying on the child order will be a strong assumption, maybe we can add a class to define the separation explicitely

File: CHANGELOG.md [hunk 3]
```diff
@@ -111,6 +111,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - [\#1640](https://github.com/cosmos/voyager/issues/1640) Hide the table proposals when there are no available ones @fedekunze
 - [\#1640](https://github.com/cosmos/voyager/issues/1640) Fixed an error that prevented the search bar to be displayed using `Ctrl+F` @fedekunze
 - Fixed testnet config build script @faboweb
+- [\#1677](https://github.com/cosmos/voyager/issues/1677) Fixed inconsistent status colors on proposals @fedekunze
```
Comment (fedekunze) [inline CHANGELOG.md]:
```suggestion
- [\#1677](https://github.com/cosmos/voyager/issues/1677) Fixed inconsistent status colors on proposals @faboweb
```

Comment (fedekunze) [reply CHANGELOG.md]: although its very nice from you to have put me as the author haha

File: app/src/renderer/components/staking/LiValidator.vue (hunk 2 above)
Comment (faboweb) [reply app/src/renderer/components/staking/LiValidator.vue]: Refactored

**Comment by faboweb** [comment]
New headers aligning left:
image

**Pull request merged**